maintenance/clis/calicoctl/install.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Install calicoctl
+description: Install the CLI for Calico.
+canonical_url: '/maintenance/clis/calicoctl/install'
+---
+
+### Big picture
+
+This guide helps you install the `calicoctl` command line tool to manage {{site.prodname}} resources 
+and perform administrative functions.
+
+### Value 
+
+The `calicoctl` command line tool is required in order to use many of {{site.prodname}}'s features. It 
+is used to manage {{site.prodname}} policies and configuration, as well as view detailed cluster status.
+
+### Concepts
+
+#### API groups
+
+All Kubernetes resources belong to an API group. The API group is indicated by the resource's `apiVersion`. For example, {{site.prodname}}
+uses resources in the `projectcalico.org/v3` API group for configuration, and the operator uses resources in the `operator.tigera.io/v1` API group.
+
+You can read more about API groups in [the Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/#api-groups).
+
+#### calicoctl and kubectl
+
+In order to manage {{site.prodname}} APIs in the `projectcalico.org/v3` API group, you should use `calicoctl`. This is because
+`calicoctl` provides important validation and defaulting for these resources that is not available in `kubectl`. However, `kubectl`
+should still be used to manage other Kubernetes resources.
+
+> **Note**: If you would like to use `kubectl` to manage `projectcalico.org/v3` API resources, you can try the 
```

Comment:
This paragraph changed to

```
> **Note**: If you would like to use `kubectl` to manage `projectcalico.org/v3` API resources, you can use the 
>           [Calico API server]({{site.baseurl}}/maintenance/install-apiserver).
{: .alert .alert-info}
```
in:

https://github.com/projectcalico/calico/blob/aceeaafdf97ca3ffb464ab77f7a3d8cf66ea4959/getting-started/clis/calicoctl/install.md

Could you make it reflect that here?

Reply:
Sure.